data/modules/DonateToCranks/DonateToCranks.lua

```diff
@@ -106,18 +106,25 @@ local onCreateBB = function (station)
 		return
 	end
 
+	local faction = Game.system.faction.name
+	local military = Game.system.faction.militaryName
+	local police = Game.system.faction.policeName
+	local system = Game.system.name
+	local stringVariables = {SYSTEM = system, FACTION = faction, MILITARY = military, POLICE = police}
 	local ad = {
 		modifier = n == 6 and 1.5 or 1.0, -- donating to FOSS is twice as good
-		title    = flavours[n].desc,
+		title    = flavours[n].title,
+		desc     = flavours[n].desc,
 		message  = flavours[n].message,
```

Comment:
It's actually easier here to store *only* `n` in the ad data and index `flavours` when placing the advertisement or generally displaying translated text to the user. :smile: The list of flavor text is not expected to change, and even when the mission is being adjusted like you're doing here it's usually only to add new flavours.

This is actually done down in onGameStart I believe; I must have simply missed cleaning it up here in onCreateBB.

Reply:
OK. Code updated... :whale:

Reply:
Looking good now!